pkg/bosh/converter/container_factory.go

```diff
@@ -272,19 +267,9 @@ func templateRenderingContainer(instanceGroupName string, secretName string) cor
 		Name:  "template-render",
 		Image: GetOperatorDockerImage(),
 		VolumeMounts: []corev1.VolumeMount{
-			{
-				Name:      VolumeRenderingDataName,
-				MountPath: VolumeRenderingDataMountPath,
-			},
-			{
-				Name:      VolumeJobsDirName,
-				MountPath: VolumeJobsDirMountPath,
-			},
-			{
-				Name:      generateVolumeName(secretName),
-				MountPath: fmt.Sprintf("/var/run/secrets/resolved-properties/%s", instanceGroupName),
-				ReadOnly:  true,
-			},
+			*renderingVolumeMount(),
+			*jobsDirVolumeMount(),
+			resolvedPropertiesVolumeMount(secretName, instanceGroupName),
```

Comment:
Why only this return copy but others return pointer?

Reply:
Because they are also used in `volume.go` and there a pointer is expected. I didn't want to introduce temporary variables just to get the reference in volume.go.

But yes, this is one of the surprising things that turned up while doing this refactoring. Sometimes we use pointers, sometimes we don't. I don't think it's bad and has to be unified now, but definitely one of the parts where this PR could be improved upon.